api/v1beta1/integrationtestscenario_suite_test.go

```diff
@@ -56,35 +63,67 @@ var _ = BeforeSuite(func() {
 
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{
-			filepath.Join("..", "..", "config", "crd", "bases"),
+		CRDDirectoryPaths:     []string{filepath.Join("..", "..", "config", "crd", "bases")},
+		ErrorIfCRDPathMissing: false,
+		WebhookInstallOptions: envtest.WebhookInstallOptions{
+			Paths: []string{filepath.Join("..", "..", "config", "webhook")},
 		},
 	}
 
-	cfg, err := testEnv.Start()
+	var err error
+	// cfg is defined in this file globally.
+	cfg, err = testEnv.Start()
 	Expect(err).NotTo(HaveOccurred())
 	Expect(cfg).NotTo(BeNil())
 
 	scheme := runtime.NewScheme()
-	Expect(AddToScheme(scheme)).To(Succeed())
+	err = AddToScheme(scheme)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
This change does the same thing but in 2 lines instead. Should we revert it?

[docs links](https://onsi.github.io/ginkgo/#suite-setup-and-cleanup-beforesuite-and-aftersuite)

-------------------
![image](https://github.com/redhat-appstudio/integration-service/assets/43273420/db997ce5-000f-48f6-a8ac-e466893f9d13)

Reply:
I copied autogenerated code, that's why.

Reply:
Reverted to original